docs/release.md

```diff
@@ -39,29 +39,59 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This is the same release cycle as Kubernetes, but KubeVirt will trail Kubernetes releases
+by about a week:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Number | KubeVirt Release Week |                     Note                    |
+|:-------------------:|:------------------:|:----------------:|-------------------------|-----------------------|:-------------------------------------------:|
+| 1                   | 1.24               | 1 (January 03)   | 0.55                    | 2 (January 10)        |                                             |
```

Comment:
Since this table is now referring to specific release versions, it might be helpful to list years associated with releases, not just months.

Reply:
+1 on what Stu said.

I even wonder if we could generalize this table, and rather maintain the release information in https://github.com/kubevirt/kubevirt/milestones

Reply:
The dates are only there for example purposes they aren't binding.  I wanted to give more context to what "week 1" is, so I'm referring to the first Monday of the year which is Jan 3 in 2022.